week-1/K-functions-parameters/exercise.js

```diff
@@ -1,9 +1,10 @@
 // Complete the function so that it takes input parameters
-function multiply() {
+function multiply(a, b) {
   // Calculate the result of the function and return it
+  return a * b;
 }
 
 // Assign the result of calling the function the variable `result`
-var result = multiply(3, 4);
+var result = multiply((a = 3), (b = 4));
```

Comment:
Hello Alisina, you function definition is very good, however your variable declaration for `result` looks problematic. instead you need to write `var result = multiply(3, 4); console.log(result);`
Thanks.

Reply:
Hi. As Kadir pointed out, you shouldn't specify the names of arguments when calling a function. If wondered why that worked, it's because assigning a value returns the value itself (a=3 returns 3 and b=4 returns 4), but you'll notice that if you changed a or b to any other name it would still work.  This is why you should use the numbers directly.